### DISCUSSION TITLE
Projects shown as "(no project)" in summary

### DISCUSSION THREAD

**Discussion by herr-carl** [mcndt/obsidian-toggl-integration#127] [Q&A] [answered]
Hello,

Thanks for making this plugin for Obsidian, an integration of toggl summary to Obsidian really helps in visualizing work status.

However, in my summary code block, all the projects are shown as "(No project)" and that makes identification of projects impossible, and donut diagram completely useless. I got 3 browser available, and all of them show the report page without problem.
![Projects shown as "No project"](https://user-images.githubusercontent.com/39486352/231701188-9f4da5b2-5f57-4800-99f6-f6c62382563f.png)

My system is Windows 10 64bit Version 22H1 (should be, up-to-date anyway). Obsidian v1.1.16 with Toggl Track Integration plugin 0.10.0 installed.

Is there anyone has the same problem? There might be some important setting that I have missed, because I'm quite new to Obsidian, but I have tried to google and search this github repository as much as I can, but still nothing useful yet, so I come here for asking help.

Hope there will be someone shed me a light.

Best,

**Comment by herr-carl** [comment, marked as answer]
Oh never mind, now the beautiful summary chart shows on my note after a rebooting. 
Thanks for making such a gorgeous plugin!
![image](https://user-images.githubusercontent.com/39486352/231971480-d79169b6-bf49-4ce0-93b2-d00ff2a5a861.png)

**Reply by mcndt** [reply]
Hey! Seems like you ran into a weird race condition where the chart managed to render before the plugin finished loading your projects. 

Please let me know if it happens again!
